pkg/cmd/pulumi/packagecmd/package_add.go

```diff
@@ -122,17 +122,17 @@ The <provider> argument can be specified in one of the following ways:
 - When <provider> is specified as a PLUGIN[@VERSION] reference, Pulumi attempts to
   resolve a resource plugin first, installing it on-demand, similarly to:
 
-    pulumi plugin install resource PLUGIN [VERSION]
+    ` + "`pulumi plugin install resource PLUGIN [VERSION]`" + `
```

Comment:
So most of the other commands are formatted like:
```
"the help text\n" +
"more help text\n" +
"help with `quotes`\n"
```
I think that's probably easier to read, and see it's correct than this splicing in of backticks in a backticked string.